### PR TITLE
PHP7.4 - Add "str_split" => "mb_str_split" mapping.

### PR DESCRIPTION
**Pull request by SpacePossum**
@ see https://wiki.php.net/rfc/mb_str_split

### CODE REVIEW

**Comment by JarJak** [comment]
This one should check first PHP version installed, no? Just got str_split replaced with mb_str_split on PHP 7.2...

File: src/Fixer/Alias/MbStrFunctionsFixer.php [hunk 1]
```diff
@@ -85,8 +86,15 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
+        $functions = array_filter(
+            self::$functionsMap,
+            static function ($mapping) {
+                return \function_exists($mapping['alternativeName']);
```
Comment (keradus) [inline src/Fixer/Alias/MbStrFunctionsFixer.php]: this was causing the issue. when we execute this, it's always truth, as we have the mb polyfill on execution time...

Comment (keradus) [reply src/Fixer/Alias/MbStrFunctionsFixer.php]: ref #4529

Comment (keradus) [reply src/Fixer/Alias/MbStrFunctionsFixer.php]: cc @SpacePossum

**Pull request merged**